tests/test_mesh_variant.cpp

```diff
@@ -0,0 +1,107 @@
+#include <catch2/catch_test_macros.hpp>
+#include <wmtk/EdgeMesh.hpp>
+#include <wmtk/PointMesh.hpp>
+
+#include <wmtk/Types.hpp>
+#include <wmtk/multimesh/utils/BasicMeshVariantRunner.hpp>
+#include <wmtk/utils/metaprogramming/as_mesh_variant.hpp>
+#include "tools/TetMesh_examples.hpp"
+#include "tools/TriMesh_examples.hpp"
+
+using namespace wmtk;
+using namespace wmtk::tests;
+
+
+namespace {
+struct DimFunctor
+{
+    // the dimension of the mesh we expect to see
+    int operator()(const PointMesh&) const
+    {
+        spdlog::info("Mesh!");
```

Comment:
I feel like we should not have any of these logs in the tests. At least not activated by default.

Reply:
Yeah. I'll delete in a new PR